android/java/res/values/brave_colors.xml

```diff
@@ -89,6 +89,17 @@
     <color name="brave_sync_bg_color">@android:color/transparent</color>
     <color name="brave_sync_border_color">@color/modern_grey_50</color>
     <color name="brave_sync_bg_secondary_color">@android:color/white</color>
+    <color name="brave_sync_stop_watch_icon_color">#0061AA</color>
```

Comment:
I did some changes at https://github.com/brave/brave-core/pull/23051/commits/bf304b5e929992aa5ab3a868b61051c2cf858731

I had top copy values from `src/brave/node_modules/@brave/leo/tokens/android/values/colors.xml`, because direct attempt to include colors
```
--- a/android/brave_java_resources.gni
+++ b/android/brave_java_resources.gni
@@ -963,6 +963,7 @@ brave_java_resources = [
   "java/res/values/array.xml",
   "java/res/values/brave_attrs.xml",
   "java/res/values/brave_colors.xml",
+  "//brave/node_modules/@brave/leo/tokens/android/values/colors.xml",
   "java/res/values/brave_dimens.xml",
```

didn't work :
```
[1/2849] ACTION //chrome/android:chrome_app_java_resources(//build/toolchain/android:android_clang_arm)
FAILED: obj/chrome/android/chrome_app_java_resources.resources.zip obj/chrome/android/chrome_app_java_resources.resources.zip.info gen/chrome/android/chrome_app_java_resources_R.txt 
python3 ../../build/android/gyp/prepare_resources.py --depfile gen/chrome/android/chrome_app_java_resources.d --res-sources-path=gen/chrome/android/chrome_app_java_resources.res.sources --resource-zip-out obj/chrome/android/chrome_app_java_resources.resources.zip --r-text-out gen/chrome/android/chrome_app_java_resources_R.txt
Error: Found files not listed in the sources list of the BUILD.gn target:
../../brave/node_modules/@brave/leo/tokens/android/drawable/ic_browser_mobile_tab_closes_app.xml
...
a lot of other files
...
```

And it looks this is the task beyond this PR.
wdyt?

Reply:
@AlexeyBarabash we need to copy/paste colors at the moment into brave_colors.xml as we don't use all of the colors mentioned in the generated file. i am going to replace existing colors with generated tokens soon.

Reply:
gotcha, thanks